static/app/components/searchQueryBuilder/useQueryBuilderGridItem.tsx

```diff
@@ -69,7 +69,10 @@ export function useQueryBuilderGridItem(
         return;
       }
 
-      const walker = getFocusableTreeWalker(e.currentTarget, {wrap: false});
+      const walker = getFocusableTreeWalker(e.currentTarget, {
+        wrap: false,
+        accept: node => node.tagName === 'BUTTON',
```

Comment:
Had to add this to make the tests happy, but I think it's also a good change. All the focusable elements we care about in a filter are buttons.